### PR TITLE
Add links to external open source communities

### PR DESCRIPTION
**Pull request by kinueng**
#### What was fixed?  (Issue # or description of fix)

Related to #2688 

#### Were the changes tested on
- [ ] Firefox (Desktop)
- [ ] Safari (Desktop)
- [x] Chrome (Desktop)
- [ ] Internet Explorer (Desktop)
- [ ] iOS (Mobile)
- [ ] Android (Mobile)
#### Running validation tools
- [ ] https://validator.w3.org/checklink
- [ ] https://validator.w3.org
- [ ] IBM Equal Access Accessibility Checker (https://www.ibm.com/able/toolkit/verify/automated)
- [ ] Lighthouse (in Chrome dev tools)

### CODE REVIEW

**Comment by kinueng** [comment]
Hi @lauracowen,
Can you help double check I got the new URL additions correct?

**Review by aknguyen7 — DISMISSED** [review]
lgtm

File: src/main/content/index.html [hunk 1]
```diff
@@ -162,26 +162,52 @@ <h2 class="develop_title">Monitor microservices in the cloud</h2>
...
                 <div class="logos_row row justify-content-center">
-                    <p>Prometheus</p>
-                    <p>Maven</p>
-                    <p>Kubernetes</p>
-                    <p>Kibana</p>
-                    <p>Elastic Search</p>
+                    <a href="https://prometheus.io/">
+                        <p>Prometheus</p>
+                    </a>
+                    <a href=" https://maven.apache.org/">
```
Comment (lauracowen) [inline src/main/content/index.html]: Random space before the URL

File: src/main/content/index.html [hunk 2]
```diff
@@ -162,26 +162,52 @@ <h2 class="develop_title">Monitor microservices in the cloud</h2>
...
+                        <p>Maven</p>
+                    </a>
+                    <a href="https://kubernetes.io/">
+                        <p>Kubernetes</p>
+                    </a>
+                    <a href=" https://www.elastic.co/kibana/">
```
Comment (lauracowen) [inline src/main/content/index.html]: Random space before URL

**Review by lauracowen — COMMENTED** [review]
The URLs look fine.
I wondered if the Grafana link should go to our OL dashboards on their site but not sure what we'd do for the other sites then though.
A couple had random spaces before the URL value that doesn't cause any problems I think but probably should be tidied up?

File: src/main/content/index.html (hunk 2 above)
Comment (kinueng) [reply src/main/content/index.html]: Resolved in commit https://github.com/OpenLiberty/openliberty.io/pull/2694/commits/4be746698399c7bac0d0ec0d5097ddbb7d3087ed

File: src/main/content/index.html (hunk 1 above)
Comment (kinueng) [reply src/main/content/index.html]: Resolved in commit https://github.com/OpenLiberty/openliberty.io/pull/2694/commits/4be746698399c7bac0d0ec0d5097ddbb7d3087ed

**Comment by kinueng** [comment]
I will get the URLs up on the draft site and have others review the links

**Review by lauracowen — APPROVED** [review]

**Pull request merged**